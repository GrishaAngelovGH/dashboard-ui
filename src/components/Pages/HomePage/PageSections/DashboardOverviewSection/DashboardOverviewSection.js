import PageSection from 'components/Pages/PageSection'
import { dashboardsOverview } from 'images'

const DashboardOverviewSection = () => (
    <PageSection color='#0b3d7f' height={520}>
        <div className='row no-gutters'>
            <div className='col-md-12 mt-2'>
                <div className='row no-gutters justify-content-center m-3'>
                    <div className='col-md-8 text-center'>
                        <h2 className='text-white'>Dashboards</h2>
                    </div>
                </div>

                <div className='row no-gutters justify-content-center mb-2 text-white text-center'>
                    <div className='col-md-8 col-11'>
                        This is a powerful, flexible and easily customizable business
                        intelligence tool for data analysis. Dashboards is the best
                        solution for displaying metrics in an understandable and readable form.
                        You may embed our dashboards into your application.
                    </div>
                </div>

                <div className='row no-gutters justify-content-center mt-3'>
                    <div className='col-md-4 col-lg-5'>
                        <img src={dashboardsOverview} className='img-fluid' />
                    </div>
                </div>
            </div>
        </div>
    </PageSection>
)

export default DashboardOverviewSection