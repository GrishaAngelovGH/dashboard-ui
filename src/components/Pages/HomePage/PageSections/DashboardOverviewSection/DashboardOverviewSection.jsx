import PageSection from 'components/Pages/PageSection'

const DashboardOverviewSection = () => (
    <PageSection style={{ background: '#0b3d7f', height: 250, alignItems: 'center' }}>
        <div className='row no-gutters'>
            <div className='col-md-12 mt-2'>
                <div className='row no-gutters justify-content-center m-3'>
                    <div className='col-md-8 text-center'>
                        <h2 className='text-white'>Dashboards</h2>
                    </div>
                </div>

                <div className='row no-gutters justify-content-center mb-2 text-white text-center'>
                    <div className='col-md-8 col-11'>
                        This is a powerful, flexible and easily customizable business
                        intelligence tool for data analysis. Dashboards is the best
                        solution for displaying metrics in an understandable and readable form.
                        You may embed our dashboards into your application.
                    </div>
                </div>
            </div>
        </div>
    </PageSection>
)

export default DashboardOverviewSection