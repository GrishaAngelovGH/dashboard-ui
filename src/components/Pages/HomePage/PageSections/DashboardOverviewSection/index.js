import DashboardOverviewSection from './DashboardOverviewSection'

export default DashboardOverviewSection