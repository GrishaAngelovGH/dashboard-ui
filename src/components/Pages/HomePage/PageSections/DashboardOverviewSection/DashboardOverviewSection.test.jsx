import { render } from '@testing-library/react'

import DashboardOverviewSection from './DashboardOverviewSection'

test('should render DashboardOverviewSection component', () => {
    const view = render(<DashboardOverviewSection />)

    expect(view).toMatchSnapshot()
})