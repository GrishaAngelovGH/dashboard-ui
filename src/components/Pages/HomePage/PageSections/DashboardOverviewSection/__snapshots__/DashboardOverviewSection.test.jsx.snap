// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render DashboardOverviewSection component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row no-gutters"
        style="background: rgb(11, 61, 127); height: 250px; align-items: center;"
      >
        <div
          class="col-md-12"
        >
          <div
            class="row no-gutters"
          >
            <div
              class="col-md-12 mt-2"
            >
              <div
                class="row no-gutters justify-content-center m-3"
              >
                <div
                  class="col-md-8 text-center"
                >
                  <h2
                    class="text-white"
                  >
                    Dashboards
                  </h2>
                </div>
              </div>
              <div
                class="row no-gutters justify-content-center mb-2 text-white text-center"
              >
                <div
                  class="col-md-8 col-11"
                >
                  This is a powerful, flexible and easily customizable business intelligence tool for data analysis. Dashboards is the best solution for displaying metrics in an understandable and readable form. You may embed our dashboards into your application.
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row no-gutters"
      style="background: rgb(11, 61, 127); height: 250px; align-items: center;"
    >
      <div
        class="col-md-12"
      >
        <div
          class="row no-gutters"
        >
          <div
            class="col-md-12 mt-2"
          >
            <div
              class="row no-gutters justify-content-center m-3"
            >
              <div
                class="col-md-8 text-center"
              >
                <h2
                  class="text-white"
                >
                  Dashboards
                </h2>
              </div>
            </div>
            <div
              class="row no-gutters justify-content-center mb-2 text-white text-center"
            >
              <div
                class="col-md-8 col-11"
              >
                This is a powerful, flexible and easily customizable business intelligence tool for data analysis. Dashboards is the best solution for displaying metrics in an understandable and readable form. You may embed our dashboards into your application.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
